### ISSUE TITLE
[ BUG ] `Get-FalconContainerVulnerability` validation of `Package` fails in PowerShell Core

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#316]
**Describe the bug**
The following error  message is produced when attempting to follow the example for [Get-FalconContainerVulnerability](https://github.com/crowdstrike/psfalcon/wiki/Get-FalconContainerVulnerability):
```powershell
Get-FalconContainerVulnerability: Cannot validate argument on parameter 'Package'. Missing property ''.
```

**Environment (please complete the following information):**
 - OS: Windows 10, Ubuntu, MacOS
 - PowerShell: 7.3.4
 - PSFalcon: 2.2.5

**Additional context**
This problem does not appear in PowerShell Desktop (i.e. PowerShell 5.1 on Windows).

**Comment by bk-cs** [comment]
This has been fixed for the 2.2.6 release.

**Comment by bk-cs** [comment]
Resolved in 2.2.6 release which has been published to GitHub and will be published on the PowerShell Gallery soon.
